src/react-components/change-scene-dialog.js

```diff
@@ -50,7 +50,7 @@ export default class ChangeSceneDialog extends Component {
               Paste a URL to a{" "}
               <IfFeature name="enable_spoke">
                 <a href="/spoke" target="_blank" rel="noopener noreferrer">
```

Comment:
i'd just change this to be a link to the phrase "custom scene" to keep it simple